libc/include/llvm-libc-macros/features-macros.h

```diff
@@ -9,6 +9,4 @@
 #ifndef LLVM_LIBC_MACROS_FEATURES_MACROS_H
 #define LLVM_LIBC_MACROS_FEATURES_MACROS_H
 
-#define __LLVM_LIBC__ 1
-
```

Comment:
perhaps we should have just deleted this file? It's literally only now empty header guards.